docs/community.rst

```diff
@@ -0,0 +1,43 @@
+=========
+Community
+=========
+
+.. todo:: 
+
+    This is just a placeholder page to explain the use and development of rasterio.
+    
+    
+Who is using rasterio?
+----------------------
+
+* `Mapbox`_ uses rasterio to process satellite imagery for it's global satellite base maps.
+* Mary Marek-Spartz's masters thesis compares the map algebra tools of the proprietary ESRI ArcPy library and the open source Rasterio library.  https://github.com/alfalimajuliett/masters-thesis
+
+Software built on rasterio
+--------------------------
+
+The following software projects use rasterio
+
+* pyimpute
+* rasterstats
+
+Rio plugins
+------------
+
+In addition to the core ``rio`` commands, visit the `Rio plugin registry`_ for a list of external plugins. 
+
+ 
+Code of conduct
+---------------
+
+.. include:: ../CODE_OF_CONDUCT.md
```

Comment:
@perrygeo this brings in a second, escaped header that reads as 
`# Contributor Code of Conduct`

And doesn't render the link at the end properly:
...` [Contributor Covenant](http://contributor-covenant.org), version 1.2.0, available at [http://contributor-covenant.org/version/1/2/0/](http://contributor-covenant.org/version/1/2/0/)`

Perhaps converting this from a markdown to a reStructuredText file would allow proper display via Github and clean inclusion here?